### DISCUSSION TITLE
First issue

### DISCUSSION THREAD

**Discussion by matosGu1lherme** [dbeaver/dbeaver#36129] [General]
Hi, i'm insterested in the project and i want to contribute, would you have any tasks for a biginners?

**Comment by E1izabeth** [comment]
Hi, @matosGu1lherme 
We have a special label, `good first issue.` Take a look at these issues, maybe you can find something https://github.com/dbeaver/dbeaver/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
